modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/AttachmentResourceTest.java

```diff
@@ -332,6 +194,77 @@ protected Attachment testDeleteAttachment_addAttachment() throws Exception {
 			_cProduct.getExternalReferenceCode(), randomAttachment());
 	}
 
+	@Override
+	protected Attachment
+			testGetProductByExternalReferenceCodeAttachmentsPage_addAttachment(
+				String externalReferenceCode, Attachment attachment)
+		throws Exception {
+
+		return attachmentResource.postProductIdAttachment(
```

Comment:
I think here is better to use the postProductByExternalReferenceCodeAttachment using the externalReferenceCode parameter and the attachment parameter